### PR TITLE
feat: Improve UI of Search Time Activity

### LINKED ISSUE #805: UI improvements in the time selection screen.
**Describe the feature you'd like**

The current UI in the time selection part of the screen is consistent. There is no differentiation between the title and the options provided in it.

**Screenshots**

time_selection

**Additional context**

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:
@iamareebjamal @nikit19 your views?

### PR DESCRIPTION
**Pull request by kshitijsuri90**
Fixes #805

Changes: 
Font changed.
CheckTextView used instead
Logic implemented so that the current choice is checked,

Screenshots for the change:
time_ui
@nikit19 @iamareebjamal Do review

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt [hunk 1]
```diff
@@ -44,6 +44,7 @@ class SearchFragment : Fragment() {
 
         rootView.timeTextView.setOnClickListener {
             val intent = Intent(activity, SearchTimeActivity::class.java)
+            intent.putExtra("value", rootView.timeTextView.text.toString())
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt]: Value? Try to use something more meaningful and make it a constant

File: app/src/main/java/org/fossasia/openevent/general/search/SearchTimeActivity.kt [hunk 2]
```diff
@@ -17,6 +17,7 @@ const val TODAY = "Today"
 const val TOMORROW = "Tomorrow"
 const val THIS_WEEKEND = "This Weekend"
 const val NEXT_MONTH = "In the next month"
+private const val SEARCHTIME: String = "time"
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/search/SearchTimeActivity.kt]: Duplication. Make the previous constant public and remove this from here. The reason we use constants is to remove duplication of writing strings which are error prone

**Comment by kshitijsuri90** [comment]
@nikit19 Do review

**Comment by kshitijsuri90** [comment]
@iamareebjamal Do merge if everything is okay. Or something more has to be done?

**Pull request merged**